terratorch/datamodules/generic_scalar_label_data_module.py

```diff
@@ -27,6 +27,21 @@ def wrap_in_compose_is_list(transform_list):
     # set check shapes to false because of the multitemporal case
     return A.Compose(transform_list, is_check_shapes=False) if isinstance(transform_list, Iterable) else transform_list
 
+def load_from_file_or_attribute(value:list[float] | str):
+
+    if type(value) == list:
+        return value
+    elif type(str): # It can be the path for a file
+        if os.path.isfile(value):
+            try:
+                content = np.genfromtxt(value).tolist()
+            except:
+                raise Exception(f"File must be txt, but received {value}")
+        else:
+            raise Exception("It seems that {value} does not exist or is not a file.")
```

Comment:
Missing f string. Also, exception messages are usually in a more imperative tone, in this case I would remove "It seems that"